packages/frontend/src/components/Form/Input.tsx

```diff
@@ -39,13 +47,20 @@ export const Input = ({ bid, setBid, isBadAmount }: InputProps) => {
         <TokenIconWrapper>
           <EtherIcon />
         </TokenIconWrapper>
-        <StyledInput value={inputValue} onChange={(e) => setValue(e.target.value)} role="input" />
+        <StyledInput
+          value={inputValue}
+          onChange={(e) => onChange(e.target.value)}
+          onBlur={(e) => onBlur(e.target.value)}
+          role="input"
+        />
```

Comment:
I'd add `placeholder="0"`, so an empty string reflects a zero amount - what do you think?